src/components/search/Search.jsx

```diff
@@ -43,27 +58,41 @@ const Search = (props) => {
     setShowAlert(props.error)
   }, [props.error])
 
+  const options = searchConfig.map(config => (<option key={config.authority} value={config.authority}>{config.label}</option>))
+
   return (
     <div id="search">
       <Header triggerEditorMenu={props.triggerHandleOffsetMenu} />
       {alert}
-      <Grid>
-        <Row className="text-center">
-          <Form>
-            <FormGroup controlId="formGroupSearch" >
-              <FormControl type="text"
-                           placeholder="Search"
-                           aria-label="Search"
-                           onChange={ event => setQueryString(event.target.value) }
-                           onKeyPress={ event => handleKeyPress(event) } />
-              <span className="help-block">Use a * to wildcard your search.</span>
-            </FormGroup>
-          </Form>
-        </Row>
-        <SearchResults {...props} key="search-results" />
-        <SearchResultsPaging {...props} pageSize="1"/>
-        <SearchResultsMessage />
-      </Grid>
+      <div className="row">
+        <form className="form-inline" onSubmit={handleSubmit}>
+          <div className="form-group col-md-4" style={{ paddingRight: '5px' }}>
+            <label htmlFor="searchType" style={{ marginRight: '5px' }}>Search</label>
+            <select className="form-control" id="searchType"
+                    value={authority} style={{ paddingRight: '0px', width: '85%' }}
```

Comment:
In an `.form-inline` aren't select's using `width: auto`?  Why are you diverging from that?